integration-tests/src/lib.rs

```diff
@@ -444,18 +507,19 @@ pub async fn delete_db(target: &postgres::Config, management_db: &str) -> Result
 /// Start service containers.
 ///
 /// Does nothing on CI where they have to be started from the outside.
-pub fn start_test_service_containers() -> Result<(), anyhow::Error> {
+pub fn start_test_service_containers() {
     static ONCE: Once = Once::new();
-    let mut res = Ok(());
     ONCE.call_once(|| {
         if !std::env::var("CI")
             .map(|value| value == "true")
             .unwrap_or_default()
         {
-            res = just(&["web-dev-up"]).map(drop);
+            if let Err(err) = just(&["web-dev-up"]) {
+                eprintln!("Can not start web-dev services: {err}");
+                abort();
+            }
```

Comment:
There is just no point in trying to run any other integration tests. This will just create a lot of verbose errors.